NullPointerException getting site title

I can´t reproduce how it is comming up, but there is a problem, if no valid siteId exist.

```
Caused by: java.lang.NullPointerException
    at com.atolcd.repo.template.EventSchedulingUtils.getSiteTitle(EventSchedulingUtils.java:64)
```

Bu you should avoid causing an NPE in this case.

```
    public String getSiteTitle(String shortName) {
        return siteService.getSite(shortName).getTitle();
    }
```

Could be changed to: 

```
    public String getSiteTitle(String shortName) {
        SiteInfo site = siteService.getSite(shortName);
        if(site != null) {
            return site.getTitle();
        }
        return "";
    }
```

Thanks.

--- a/module_alfresco/src/java/com/atolcd/repo/jscript/EventSchedulingUtils.java
+++ b/module_alfresco/src/java/com/atolcd/repo/jscript/EventSchedulingUtils.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Atol Conseils et Developpements.
+ * Copyright (C) 2012 Atol Conseils et Développements.
  * http://www.atolcd.com/
  * Author: Bertrand FOREST
  *

--- a/module_alfresco/src/java/com/atolcd/repo/jscript/EventSchedulingUtils.java
+++ b/module_alfresco/src/java/com/atolcd/repo/jscript/EventSchedulingUtils.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Atol Conseils et Développements.
+ * Copyright (C) 2012 Atol Conseils et Developpements.
  * http://www.atolcd.com/
  * Author: Bertrand FOREST
  *

--- a/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
+++ b/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Atol Conseils et Developpements.
+ * Copyright (C) 2012 Atol Conseils et Développements.
  * http://www.atolcd.com/
  * Author: Bertrand FOREST
  *

--- a/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
+++ b/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2012 Atol Conseils et Développements.
+ * Copyright (C) 2012 Atol Conseils et Developpements.
  * http://www.atolcd.com/
  * Author: Bertrand FOREST
  *
@@ -28,6 +28,7 @@ import org.alfresco.service.cmr.site.SiteService;
 import org.springframework.beans.factory.InitializingBean;
 import org.springframework.extensions.surf.util.I18NUtil;
 import org.springframework.util.Assert;
+import org.alfresco.service.cmr.site.SiteInfo;
 
 public class EventSchedulingUtils extends BaseTemplateProcessorExtension implements InitializingBean {
 	private NodeService nodeService;
@@ -61,7 +62,11 @@ public class EventSchedulingUtils extends BaseTemplateProcessorExtension impleme
 	}
 
 	public String getSiteTitle(String shortName) {
-		return siteService.getSite(shortName).getTitle();
+		if(shortName == null) {
+			return null;
+		}
+		SiteInfo siteInfo = siteService.getSite(shortName);
+		return siteInfo != null ? siteInfo.getTitle() : null;
 	}
 
 	public String getPersonFullName(String username) {

--- a/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
+++ b/module_alfresco/src/java/com/atolcd/repo/template/EventSchedulingUtils.java
@@ -62,7 +62,7 @@ public class EventSchedulingUtils extends BaseTemplateProcessorExtension impleme
 	}
 
 	public String getSiteTitle(String shortName) {
-		if(shortName == null) {
+		if(shortName == null || shortName.isEmpty()) {
 			return null;
 		}
 		SiteInfo siteInfo = siteService.getSite(shortName);

